### ISSUE TITLE
how can i fix this problem

### ISSUE THREAD

**Issue by 08star** [python-telegram-bot/python-telegram-bot#3160]
### Issue I am facing

AttributeError: type object 'ContextTypes' has no attribute 'DEFAULT_TYPE'

### Operating System

w10

### Version of Python, python-telegram-bot & dependencies

```shell
python-telegram-bot 20.0a0
Bot API 6.0
Python 3.10.5 (tags/v3.10.5:f377153, Jun  6 2022, 16:14:13) [MSC v.1929 64 bit (AMD64)]
```

**Comment by harshil21** [comment]
run pip install python-telegram-bot -U --pre. That is only available from 20.0a1. Also checkout the [release notes](t;.me/pythontelegrambotchannel) for the changes.

**Comment by 08star** [comment]
Thank you, really solved the problem
